web/satellite/src/components/navigation/NavigationArea.vue

```diff
@@ -246,6 +246,7 @@ export default class NavigationArea extends Vue {
     public navigateToCreateAG(): void {
         this.analytics.eventTriggered(AnalyticsEvent.CREATE_AN_ACCESS_GRANT_CLICKED);
         this.closeDropdowns();
+        this.analytics.pageVisit(RouteConfig.AccessGrants.with(RouteConfig.CreateAccessGrant).with(RouteConfig.NameStep).path);
```

Comment:
why are you adding NameStep to the path here?

Reply:
because when the popup opens, it should send this link /access-grants/create-grant/name. But if I write this "RouteConfig.AccessGrants.with(RouteConfig.CreateAccessGrant).path", it only sends "/access-grants/create-grant"